Update README.md (#13)
vtex-apps/buybox-context: docs/README.md

  | `price` | `string` | Sort sellers **only product price**. The sellers at the top of the list will have the lowest prices. |
  | `priceShipping` | `string` | Sort sellers by **product price + shipping price**. The lowest sum between these two values, will be positioned at the top of the sellers list. |
  | `customExpression` | `string` | Sort sellers by a custom expression based on a math expression, with predefined [`variables`](#custom-expression-variables). The lowest value resultant of this expression, will be positioned at the top of the sellers list. For more details about how to create expressions, click [here](https://github.com/silentmatt/expr-eval#readme) |

>   | `protocol` | `string` | Sort sellers using a protocol defined on [buybox-graphql](https://github.com/vtex-apps/buybox-graphql). This protocol defines the contract to receive and response data. So it works, it's necessary to implement a resolver for this contracts. In this case, was created the [buybox-resolver](https://github.com/vtex-apps/buybox-resolver) as example. |

Reviewer: ```suggestion
  | `protocol` | `string` | Sort sellers using a protocol defined on [buybox-graphql](https://github.com/vtex-apps/buybox-graphql). This protocol defines the contract to receive and respond the sorted data. For that to work, it's necessary to implement a resolver for this contract. The [buybox-resolver](https://github.com/vtex-apps/buybox-resolver) was created as an example of that implamentation. |
```
